src/Meshes/abstractMesh.ts

```diff
@@ -663,8 +663,8 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
 
     /** @hidden */
     public _masterMesh: Nullable<AbstractMesh> = null;
-    /** @hidden */
-    public _boundingInfo: Nullable<BoundingInfo> = null;
+    private _boundingInfo: Nullable<BoundingInfo> = null;
```

Comment:
Hi @deltakosh 
Would it be possible to turn these 2 props back to public ?

They are hidden and they are the only private props in this class. And migrating to BabylonJs 5 breaks my code :)
Cheers

Reply:
(public or protected at least)

Reply:
what broke your code? (You were using _features right? :))

Reply:
I have my own derivation of `Mesh` that overwrites `_refreshBoundingInfo` and then accesses and sets `_boundingInfo`.

The error message is "Property '_boundingInfo' is private and only accessible within class 'AbstractMesh'."

Reply:
(of course I can work-around that with some ugly `any`)

Reply:
No worries to flag it as protected:) wanna do a PR?